lib/MySQL_Session.cpp

```diff
@@ -2863,14 +2859,16 @@ bool MySQL_Session::handler_again___status_CHANGING_AUTOCOMMIT(int *_rc) {
 				// client error, serious
 				if (myerr != 0) {
 					proxy_error("Detected a broken connection during SET AUTOCOMMIT on %s , %d : %d, %s\n", myconn->parent->address, myconn->parent->port, myerr, mysql_error(myconn->mysql));
+					MyHGM->p_update_mysql_error_counter(p_mysql_error_type::mysql, myconn->parent->myhgc->hid, myconn->parent->address, myconn->parent->port, myerr);
 				} else {
 					proxy_error(
 						"Detected a broken connection during SET AUTOCOMMIT on %s , %d : %d, %s\n",
 						myconn->parent->address,
 						myconn->parent->port,
-						myerr,
+						ER_PROXYSQL_OFFLINE_SRV,
```

Comment:
This can stay `myerr` .
For the same replace didn't happen in othe functions, like `handler_again___status_SETTING_INIT_CONNECT`